src/gimelstudio/core/project_file.py

```diff
@@ -144,8 +148,11 @@ def WriteFile(self, file_path, contents):
 
     def OpenFile(self, file_path):
         with open(file_path, "r") as file:
-            file_contents = json.load(file)
-            self.contents = file_contents
+            try:
+                file_contents = json.load(file)
+                self.contents = file_contents
+            except:
+                wx.MessageBox(message=_("Unable to create graph from file, the file may be corrupt\n(Details: check 2 failed to complete:)"))
```

Comment:
``Unable to open project from file. Opening the file failed. The file may be corrupted.`` 

My experience is that unless you have documented error codes, writing something like ``check 2 failed to complete`` isn't all that helpful. It's better to just state what went wrong. 🙂